bot/exts/evergreen/pythonfacts.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+import discord
+from discord.ext import commands
+
+
+with open('bot/resources/evergreen/python_facts.txt') as file:
+    FACTS = list(file)
+
+
+class PythonFacts(commands.Cog):
+    """Gives a random fun fact about Python."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.command(name='pythonfact', aliases=['pyfact'])
+    async def get_python_fact(self, ctx: commands.Context) -> None:
+        """Gives a Random fun fact about Python."""
+        await ctx.send(embed=discord.Embed(title='Python Facts', description=f'**{random.choice(FACTS)}**'))
```

Comment:
```suggestion
        await ctx.send(embed=discord.Embed(title='Python Facts', description=random.choice(FACTS)))
```
No need of bold in the description.